packages/dataviews/src/normalize-fields.ts

```diff
@@ -56,16 +58,14 @@ export function normalizeFields< Item >(
 
 		const Edit = getControl( field, fieldTypeDefinition );
 
-		const renderFromElements = ( { item }: { item: Item } ) => {
```

Comment:
This has been moved to field type definitions, including the "untyped" one, so it works as before.